scripts/conflict_detector.py

```
@@ -0,0 +1,495 @@
+                "high_severity": sum(1 for c in self.conflicts if c.severity == "high"),
+                "medium_severity": sum(1 for c in self.conflicts if c.severity == "medium"),
+                "low_severity": sum(1 for c in self.conflicts if c.severity == "low"),
+                "files_to_remove": sum(len(c.conflicting_files) for c in self.conflicts)
+            }
+        }
+        return json.dumps(data, indent=2, ensure_ascii=False)
+    
+    def get_files_to_remove(self, severity_filter: Optional[str] = None) -> List[str]:
+        """获取建议删除的文件列表"""
+        files_to_remove = []
+        severity_order = {"high": 0, "medium": 1, "low": 2}
```

Ruff (0.11.9)

Local variable `severity_order` is assigned to but never used

Remove assignment to unused variable `severity_order`

(F841)